### PR TITLE
Add more splashes.

### PR DESCRIPTION
**Pull request by drXor**

### CODE REVIEW

File: resources/assets/minecraft/texts/splashes.txt [hunk 1]
```diff
@@ -51,3 +58,12 @@ Ceci n'est pas un splash.
 Xor is not actually a cat.
 The MD5 of md_5 is e14cfacdd442a953343ebd8529138680
 I blame Scetch for this!
+Burritos are superior.
```
Comment (Zbob750) [inline resources/assets/minecraft/texts/splashes.txt]: What is this heresy

Comment (Cldfire) [inline resources/assets/minecraft/texts/splashes.txt]: Heresy or not, someone has to tell stand up for our rights. #burritonation

Comment (Zbob750) [inline resources/assets/minecraft/texts/splashes.txt]: #burritosaregits

**Comment by phase** [comment]
Please fill this with cringe memes.

**Comment by Thinkofname** [comment]
Sure, why not

File: resources/assets/minecraft/texts/splashes.txt [hunk 2]
```diff
@@ -51,3 +58,12 @@ Ceci n'est pas un splash.
...
+Z750 is a living taco.
+sudo chmod -x `which chmod`
+java.lang.OutOfMemoryError: failed to allocate java.lang.OutOfMemoryError
+define_blocks! { ... }
+macros_rule!
+cat /dev/random > steven; git push --force
```
Comment (Techcable) [inline resources/assets/minecraft/texts/splashes.txt]: force push is evil

Comment (phase) [inline resources/assets/minecraft/texts/splashes.txt]: He needs to commit it before he pushes

**Pull request merged**